### ISSUE TITLE
RE2 Claire A Base Vaccine Appearing in Factory

### ISSUE THREAD

**Issue by PlantBrain** [IntelOrca/biorand#594]
### Version

3.1.3 (cbe02d7)

### Seed

R721-ZREK-XEPH40AZ9R000KJRZ3ZZZZ1XXZQZZVVZQZZZH0Y50CZZ

### Scenario

RE 2 (Claire A)

### Player Character

Claire (RE2)

### What happened?

This seed has segmented placement checked and the Base Vaccine appeared in the Factory, room 501:2. Isn't this item supposed to only appear in the lab section?

[log_pl1.txt](https://github.com/IntelOrca/biorand/files/12607562/log_pl1.txt)

**Comment by IntelOrca** [comment]
Yeah, this is a glitch due to the odd way in which the original weapon key is in a different segment to where it is required. I thought I had fixed this, but I guess not. Thanks for reporting.

**Comment by IntelOrca** [comment]
Temporarily resolved by making item low priority
